pom.xml

```diff
@@ -72,7 +82,7 @@
       <dependency>
           <groupId>org.jenkins-ci.plugins</groupId>
           <artifactId>git</artifactId>
-          <version>3.0.0</version>
+          <version>4.0.0-rc3002.35b0975404eb</version>
```

Comment:
I assume that change will be reverted before release, since you don't want to require users of the stash plugin to install a pre-release of the git plugin

Reply:
Yes, this is just a way to verify the change. I should have been more explicit about it